src/d3/hapticplot.d3.ts

```diff
@@ -83,7 +86,19 @@ export class Hapticplot{
   }
 
   /**
-   * Triggers a haptic pulse, changes a points color and size when a it is hovered by the controller entity
+   * Attaches audio triggers to each point, mapping associated data to mp3 marimba notes
+   * @param datum Ingested Data
+   * @param point the point who's position is being set
+   */
+  private setAudio(datum, point){
```

Comment:
I noticed while testing this that hitting the same point several times in a row doesn't replay the same noise unless something amount of time has elapsed.  Is this intentional?

Reply:
It isn't. I think that is coming from the fact that the mp3 files have ~1 second a quite reverb at the end, and calling play on one of them while it is already playing won't interrupt it. I think a good work around is checking if a sound is playing before playing it, and stopping it if it is.